projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-row.component.html

```diff
@@ -98,15 +98,25 @@
 
 
     <ng-template #headerTemplate let-column>
-        <div class="igx-grid__tr--header">
-            <igx-icon [attr.draggable]="false" (click)="grid.toggleRow(getRowDimensionKey(column))">
-                      {{ getExpandState(column) ? 'expand_more' : 'expand_less'}}</igx-icon>
+        <div class='igx-grid__tr--header igx-grid__row-indentation--level-{{getLevel(column)}}'>
+            <igx-icon [attr.draggable]="false"
+                      (click)="grid.toggleRow(getRowDimensionKey(column))">
+                      {{ getExpandState(column) ? 'expand_more' : 'chevron_right'}}</igx-icon>
             {{column.header}}
         </div>
     </ng-template>
 
     <ng-template #headerTemplateGray let-column>
-        <div class="igx-grid__tr--header">
+        <div class='igx-grid__tr--header igx-grid__row-indentation--level-{{getLevel(column)}}'>
```

Comment:
Do we still need both default template and gray template ?

Reply:
I guess this comment is for @desig9stein and @StefanIvanov. Should we drop completely gray styles before having the cell merging feature?

Reply:
We can drop it for now, this version is just a preview, so let's not add things that we wouldn't like to be there later on if this is the case here :)

Reply:
Ok. In that case I'll remove the whole template.